tests/api/ask_astro/rest/controllers/test_list_recent_requests.py

```diff
@@ -0,0 +1,58 @@
+from unittest.mock import AsyncMock, Mock, PropertyMock, patch
+
+import pytest
+from sanic import Sanic
+from sanic_testing import TestManager
+
+sanitized_name = __name__.replace(".", "_")
+
+
+@pytest.fixture
```

Comment:
Should we move it to confest.py ?

Reply:
https://github.com/astronomer/ask-astro/pull/77/commits/9d6b18e5922b9e6ac4900f772c57257e76086fc9